spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -230,14 +255,42 @@ def show_total_results(self):
         Print total results to CLI
         """
 
-        printv('\n\nAveraged measures...', self.verbose, 'normal')
+        printv('\n\nAveraged measures across all lesions...', self.verbose, 'normal')
 
         for key in self.measure_keys:
             mean_value = np.round(np.mean(self.measure_pd[key]), 2)
             std_value = np.round(np.std(self.measure_pd[key]), 2)
             measure_info = f'  {key} = {mean_value} +/- {std_value}'
             printv(measure_info, self.verbose, type='info')
 
+        # For the tissue bridges, we get the minimum bridges across all lesions for the midsagittal slice
+        if self.fname_sc is not None:
+            printv('\nMinimum tissue bridges across all lesions for the midsagittal slice...', self.verbose, 'normal')
+            midsagittal_dorsal_bridges = list()
+            midsagittal_ventral_bridges = list()
+            # Iterate across lesions to get the bridges for the midsagittal slice
+            for idx, row in self.measure_pd.iterrows():
+                if row['midsagittal_spinal_cord_slice'] is not None:        # just safety check
+                    # Get the midsagittal slice number for the selected lesion
+                    # Note that the midsagittal slice is the same for all lesions as it is based on the spinal cord
+                    # segmentation
+                    midsagittal_slice = str(int(row['midsagittal_spinal_cord_slice']))
+                    # Get dorsal and ventral tissue bridges for the mid-sagittal slice
+                    dorsal_tissue_bridge = row['slice_' + midsagittal_slice + '_dorsal_bridge_width [mm]']
+                    ventral_tissue_bridge = row['slice_' + midsagittal_slice + '_ventral_bridge_width [mm]']
```

Comment:
Nitpick: these could be f-strings.
```suggestion
                    dorsal_tissue_bridge = row[f'slice_{midsagittal_slice}_dorsal_bridge_width [mm]']
                    ventral_tissue_bridge = row[f'slice_{midsagittal_slice}_ventral_bridge_width [mm]']
```